changelog.d/798.removal

```diff
@@ -1 +1 @@
-Drop support for the Sled crypto store format. Users must disable/remove the configuration key of `experimentalEncryption.useLegacySledStore`, and the crypto store will always use the SQLite format. If an existing SQLite store does not exist on bridge startup, one will be created.
+Drop support for the Sled crypto store format. The configuration key of `experimentalEncryption.useLegacySledStore` is now ignored, and the crypto store will always use the SQLite format. If an existing SQLite store does not exist on bridge startup, one will be created.
```

Comment:
This is an old changelog from an incomplete version of #798.  The removal of `useLegacySledStore` in `Config.ts` also comes from an incomplete version of that PR.

Now that #798 has been submitted, this PR can be rebased onto the tip of `main` instead of being a merge.